scripts/postalcode.cs.template

```diff
@@ -0,0 +1,102 @@
+using System;
+using System.Text.RegularExpressions;
+using PostalCodes.GenericPostalCodes;
+
+namespace PostalCodes
+{
+	internal class @@CountryCode@@PostalCode : AlphaNumericPostalCode
```

Comment:
It would be cool if the generated classes were partial classes. It would always make it easier to add extra overridden (non generated) functionality to the exotic zip codes.